src/audits/accessibility/aria-valid-attr.js

```diff
@@ -43,6 +43,15 @@ class ARIAValidAttr extends Audit {
   }
 
   /**
+   * @return {!Array<string>}
```

Comment:
ha, I actually prefer the nicely spaced version. Easy to see it's an array, etc. The second isn't the worst, though.